rider/src/main/resources/META-INF/plugin.xml

```diff
@@ -184,6 +181,10 @@
             use-shortcut-of="ShowIntentionActions">
       <add-to-group group-id="SolutionExplorerPopupMenu" anchor="last" />
     </action>
+
+    <action id="InstallEditorPluginAction"
+            class="com.jetbrains.rider.plugins.unity.actions.InstallEditorPluginAction"
+            text="Install Unity Editor Plugin" />
```

Comment:
Would it be better to leave this plugin where it was, but add some logic in `update` to hide it if Unity is 2019.2+? So it would still be available for older versions of Unity, but hidden when it's not needed.

Reply:
Someone from the test team told me that this action is confusing and is not actually useful for anyone. I started to think about that and decided to remove it from the Menu.